lib/arel/nodes/unary_operation.rb

```diff
@@ -0,0 +1,25 @@
+module Arel
+  module Nodes
+
```

Comment:
@rafaelfranca this blank line follows the pattern in infix_operation.rb.  Did you want me to remove it from there as well or just leave it here?